AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/TestSupport/HubEventsIntegrationTestBase.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class HubEventsIntegrationTestBase: XCTestCase {
```

Comment:
why not reuse `SyncEngineIntegrationTestBase`?

Reply:
synced offline, sounds like we need a simplified version of `SyncEngineIntegrationTestBase`. then i suggest additionally, if your Hub events is so closely coupled to the start of DataStore (Amplify.configure()), you may want to hoist the call to `Amplify.configure()` out of the test base's setup method and into your test. Then you'll be able to clearly write a test comment describing the test
```
Given: DataStore with sync enabled
When: `Amplify.configure()`  to start DataStore and the sync engine
Then: The events [X, Y, Z] are emitted as part of the sync engine lifecycle
```

because right now your `Amplify.configure()` is hidden inside the testbase